winlogbeat/eventlog/wineventlog_test.go

```diff
@@ -189,6 +190,25 @@ func testWindowsEventLog(t *testing.T, api string) {
 		return openLog(t, api, nil, config)
 	}
 
+	t.Run("has_message", func(t *testing.T) {
+		log := openLog(t, map[string]interface{}{"name": providerName, "batch_read_size": 1})
+		defer log.Close()
+
+		for i := 0; i < 10; i++ {
+			records, err := log.Read()
+			require.NotEmpty(t, records)
+			require.NoError(t, err)
+			if i == 0 {
+				// The first event in a collection of events created by eventcreate
```

Comment:
IIRC the modulo on L186 is bad. The eventcreate event ID values should range on [1-1000] instead of [0-1000]. Saw this too when I was hacking.

> Specifies the event ID for the event. A valid ID is any number from 1 to 1000.

https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/eventcreate